src/main/java/co/omise/ConditionUtility.java

```diff
@@ -0,0 +1,22 @@
+package co.omise;
+
+
+/**
+ * A utility class that encases methods for checking certain conditions, for example a object not being null
+ */
+public class ConditionUtility {
```

Comment:
`Precondition` ?

Reply:
It is essentially what this is but didn't want to down right copy the name. We're probably gonna remove this anyway I think